src/components/CharacterModal.jsx

```diff
@@ -52,25 +55,25 @@ export default function CharacterModal(props) {
       onOk={onModalOk}
       onCancel={handleCancel}
       footer={[
-        <Button key="back" onClick={handleCancel}>
-          Cancel
+        <Button key='back' onClick={handleCancel}>
+          {t('cancel')}
         </Button>,
-        <Button key="submit" type="primary" onClick={onModalOk}>
-          Save
+        <Button key='submit' type='primary' onClick={onModalOk}>
+          {t('save')}
```

Comment:
Is there a reason these strings are t('lowercase') instead of t('Uppercase')? Feels like this might get confusing at some point

Reply:
mixed case is handled by a "capitalize" formatter, so the keys can be case insensitive